README.md

```diff
@@ -61,21 +61,26 @@ Download the lastest version of Android Studio at the following link: http://dev
 
 ## Javadoc
-JavaDoc Link: http://ucsb-cs56-projects.github.io/cs56-android-conversion-showdown/javadoc/
+JavaDoc Link: https://github.com/kerrymo/cs56-android-conversion-showdown/tree/proj01/javadoc
+
+
 
 ## Screenshots
 
 Screenshots of the running Application goes here. <br />
-![Alt tag](https://github.com/henryhyu/android-quiz/blob/master/EntryScreen.PNG)
-![Alt tag](https://github.com/henryhyu/android-quiz/blob/master/WelcomeScreen.PNG)
-![Alt tag](https://github.com/henryhyu/android-quiz/blob/master/Calculator.PNG)
-![Alt tag](https://github.com/henryhyu/android-quiz/blob/master/TestSelection.PNG)
-![Alt tag](https://github.com/henryhyu/android-quiz/blob/master/QuizBasic.PNG)
-![Alt tag](https://github.com/henryhyu/android-quiz/blob/master/QuizBinaryE.PNG)
-![Alt tag](https://github.com/henryhyu/android-quiz/blob/master/QuizOctalE.PNG)
-![Alt tag](https://github.com/henryhyu/android-quiz/blob/master/QuizIncorrect.PNG)
-![Alt tag](https://github.com/henryhyu/android-quiz/blob/master/ResultBad.PNG)
-![Alt tag](https://github.com/henryhyu/android-quiz/blob/master/ResultGood.PNG)
+![Alt tag](https://github.com/kerrymo/cs56-android-conversion-showdown/blob/proj01/app/src/readmePic/ss0.png)
+![Alt tag](https://github.com/kerrymo/cs56-android-conversion-showdown/blob/proj01/app/src/readmePic/ss1.png)
+![Alt tag](https://github.com/kerrymo/cs56-android-conversion-showdown/blob/proj01/app/src/readmePic/ss2.png)
+![Alt tag](https://github.com/kerrymo/cs56-android-conversion-showdown/blob/proj01/app/src/readmePic/ss3.png)
+![Alt tag](https://github.com/kerrymo/cs56-android-conversion-showdown/blob/proj01/app/src/readmePic/ss4.png)
+![Alt tag](https://github.com/kerrymo/cs56-android-conversion-showdown/blob/proj01/app/src/readmePic/ss5.png)
+![Alt tag](https://github.com/kerrymo/cs56-android-conversion-showdown/blob/proj01/app/src/readmePic/ss6.png)
+![Alt tag](https://github.com/kerrymo/cs56-android-conversion-showdown/blob/proj01/app/src/readmePic/ss7.png)
+![Alt tag](https://github.com/kerrymo/cs56-android-conversion-showdown/blob/proj01/app/src/readmePic/ss8.png)
+![Alt tag](https://github.com/kerrymo/cs56-android-conversion-showdown/blob/proj01/app/src/readmePic/ss9.png)
+![Alt tag](https://github.com/kerrymo/cs56-android-conversion-showdown/blob/proj01/app/src/readmePic/ss10.png)
+![Alt tag](https://github.com/kerrymo/cs56-android-conversion-showdown/blob/proj01/app/src/readmePic/ss11.png)
+![Alt tag](https://github.com/kerrymo/cs56-android-conversion-showdown/blob/proj01/app/src/readmePic/ss12.png)
```

Comment:
convert these links to relative instead of absolute - you can use just `/app/src/readmePic/ss#.png`

Reply:
Fixed